.github/workflows/olm.yml

```diff
@@ -0,0 +1,47 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  main:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        go: [ '1.15' ]
+    name: Go ${{ matrix.go }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      id: go
+      with:
+        go-version: ${{ matrix.go }}
+
+    - name: Create and set up K8s KinD Cluster
+      run: |
+        kind delete cluster
+        ./scripts/kind-with-registry.sh
+        make kind-deploy-olm
+
+    - name: Build and Push Test Container Image to KIND node
+      run: |
+        make docker-build IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
+        make docker-push IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
+        make bundle IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
+        make bundle-build REPO=localhost:5000 VERSION=$GITHUB_SHA
+        make docker-push IMG=localhost:5000/gatekeeper-operator-bundle:$GITHUB_SHA
+        make bundle-index-build REPO=localhost:5000 VERSION=$GITHUB_SHA
+        make docker-push IMG=localhost:5000/gatekeeper-operator-bundle-index:$GITHUB_SHA
+
+    - name: Deploy resources on KIND cluster to install Gatekeeper
+      run: |
+        make set-olm-e2e-img INDEX_IMAGE=localhost:5000/gatekeeper-operator-bundle-index:$GITHUB_SHA
+        kubectl apply -f config/olm-install/install-resources.yaml
```

Comment:
We should add this command as part of the make recipe so users can use make to execute it. See comment below.